clients/java/signalr/src/main/java/com/microsoft/signalr/WebsocketWrapperOnClosed.java

```diff
@@ -0,0 +1,8 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+package com.microsoft.signalr;
+
+interface WebsocketWrapperOnClosed {
```

Comment:
Upper case `S` and there's no need to include `Wrapper` in this type even though it's used by the `WebSocketWrapper` type (it's still triggered because the WebSocket closed, so the name only needs to reflect that). And I think it should be a little more aligned with our other callback types with the `Callback` suffix:

```suggestion
interface WebSocketOnClosedCallback {
```